src/components/Navbar/Navbar.tsx

```diff
@@ -0,0 +1,16 @@
+import { useMediaQuery } from '@mantine/hooks';
+import { theme } from '@/theme';
+import DesktopNav from './DesktopNav';
+import MobileNav from './MobileNav';
+
+export const NAVIGATION_LINKS = [
+  { label: 'Home', href: '/' },

```

Comment:
Both labels and href should come from constant file (check the router file, its already using existing `ROUTES` constants )